### PR TITLE
Fix incorrect description for `randomPrice` upper limit

### PR DESCRIPTION
**Pull request by coditva**
Ref https://github.com/postmanlabs/postman-collection/pull/1208#discussion_r682686137

### CODE REVIEW

File: lib/superstring/dynamic-variables.js [hunk 1]
```diff
@@ -312,7 +312,7 @@ var faker = require('faker/locale/en'),
             generator: faker.finance.mask
         },
         $randomPrice: {
-            description: 'A random price between 0.00 and 999.00',
+            description: 'A random price between 0.00 and 999.99',
```
Comment (neilstudd) [inline lib/superstring/dynamic-variables.js:315]: I promise this needs to be `1000.00`, not `999.99` 😃 

Demo from Tests tab

```
prices = []
for (i = 1; i < 100000; i++) {
    prices.push(pm.variables.replaceIn('{{$randomPrice}}'));
}
prices.sort((function(a, b){return a-b}));
console.log("Prices range from " + prices[0] + " to " + prices[prices.length-1]);
```

Results in console message `Prices range from 0.00 to 1000.00`

**Review by neilstudd — CHANGES_REQUESTED** [review]
Made one small ($0.01) change request :)

File: lib/superstring/dynamic-variables.js (hunk 1 above)
Comment (coditva) [reply lib/superstring/dynamic-variables.js:315]: @neilstudd I tried this but could only find the max price to be `999.99` 🤔 
```javascript
const dynamicVars = require('./lib/superstring/dynamic-variables');

let max = 0.00;

for (let i = 0, len = 10000000; i < len; i++) {
    const price = dynamicVars.$randomPrice.generator();

    // console.log(price);

    if (max < price) {
        max = price;
    }
}

console.log(max);
```

Let me try the script you shared as well.

Comment (neilstudd) [reply lib/superstring/dynamic-variables.js:315]: Alternate targeted example:

```
for (i = 1; i < 100000; i++) {
    if ( pm.variables.replaceIn('{{$randomPrice}}') == "1000.00") {
        console.log("found 1000");
    }
}
```

Comment (coditva) [reply lib/superstring/dynamic-variables.js:315]: @neilstudd Aah, makes sense. 🤦🏼
Let me update this.

**Review by neilstudd — APPROVED** [review]

**Review by codenirvana — APPROVED** [review]

**Pull request merged**